students/cocokaku/lesson6/test_mailroom.py

```diff
@@ -0,0 +1,185 @@
+#!/usr/bin/python3
+"""
+mailroom part 4: adding unit testing
+
+modules to test:
+    list_donors(db)
+    add_donation(db, name, amount)
+    thank_you_letter(db, name)
+    summary_report(db)
+    get_second(elem)
+    
+modules not tested:
+    send_thank_you(db) - user input
+    send_all_letters(db) - user input
+    create_a_report(db) - no logic
+    main_menu_error(_) - no logic
+    quit_program(_) - no logic
+"""
+
+from mailroom import list_donors
+from mailroom import add_donation
+from mailroom import thank_you_letter
+from mailroom import summary_report
+from mailroom import get_second
+
+
+# test list_donors(db) - 4 tests
+def test_list_donors_empty_db():
+    db = {}
+    res_string = ""
+    assert list_donors(db) == res_string

```

Comment:
it is common to use variable called `actual` to represent that this is the thing you expect - it's pretty obvious in this small test but may be helpful in a bit more complex tests, for example:
```
def test_list_donors_empty_db():
    db = {}
    expected = ""
    assert list_donors(db) == expected
```